### DISCUSSION TITLE
图片 gallery 随机生效

### DISCUSSION THREAD

**Discussion by OceanRC** [CaiJimmy/hugo-theme-stack#892] [Q&A] [answered]
图片全部与 index.md 在一个文件夹，在本机上预览和 github pages 上效果一样。

这是我的markdown原文：
![这是markdown原文](https://github.com/CaiJimmy/hugo-theme-stack/assets/145827067/d6563d21-f039-4835-9e53-c9cd494a3149)

这是我的网站表现出来的效果：
![这是网站效果](https://github.com/CaiJimmy/hugo-theme-stack/assets/145827067/66a4266b-8a27-4aa8-b1f7-d7522c393630)

项目地址：https://github.com/OceanRC/oceanRC.github.io 与 Github Pages：[https://oceanRC.github.io](https://oceanrc.github.io./p/%E5%A6%82%E4%BD%95%E5%9C%A8-windows-terminal-%E4%B8%AD%E4%BD%BF%E7%94%A8-git-bash/)。

我不太理解为什么三行一样的图片，怎么只有前两个使用 gallery 的……

是不是因为这个的原因？
![图片](https://github.com/CaiJimmy/hugo-theme-stack/assets/145827067/1a72e7d7-2551-4d02-905d-1a2bd724bc78)

**Comment by CaiJimmy** [comment, marked as answer]
大概是因为你文章里面有 Zero Width Space，删掉应该就好了。

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/5889006/9b10c6cb-eccb-4eaf-a9dd-2615bd0adfca)

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/5889006/912c3400-e30e-44d8-9fd0-7321e538050f)
